packages/vite/src/node/build.ts

```diff
@@ -556,6 +561,8 @@ async function doBuild(
       prepareOutDir(outDir, options.emptyOutDir, config)
     }
 
+    // clear cache for parallel builds
+    cacheVendorChunks.clear()
```

Comment:
Please focus on checking whether it needs to be cleared here?
Will the parallelCallCounts introduced by 4a955bc actually be greater than 1? I tested several scenarios that I expected and found nothing. If it is greater than, maybe parallel build will affect the shared cache?